201/ReadMe

```diff
@@ -0,0 +1,85 @@
+
+# 201 sample
+
+This sample shows you how to use utilize Azure Key Vault, Azure Container Registry and Azure Arc enabled Key Vault Extension to dynamically pull ACR credentials from AKV and pull image from ACR.
+## Prerequisites
+* Compatible cluster
```

Comment:
Do we need to list a few examples or standards for compatible cluster here? Otherwise it may be ambiguous about "Compatible"

Reply:
Let's wait on Peter's input on this.

Reply:
I'm inclined to get it in now and I'll spend quality time to pretend that I didn't read any of these.

Reply:
On this particular matter, maybe add a recommendation or something would be good enough.